WebKitBrowser/WebKitBrowser.conf.in

```diff
@@ -26,6 +26,8 @@ configuration.add("msebuffers", "@PLUGIN_WEBKITBROWSER_MSEBUFFERS@")
 configuration.add("thunderdecryptorpreference", "@PLUGIN_WEBKITBROWSER_THUNDER_DECRYPTOR_PREFERENCE@")
 configuration.add("memoryprofile", "@PLUGIN_WEBKITBROWSER_MEMORYPROFILE@")
 configuration.add("memorypressure", "@PLUGIN_WEBKITBROWSER_MEMORYPRESSURE@")
```

Comment:
let also add a "if(WEBKIT_MEMORY_PRESSURE_API)" (but of course one that is correct for python :) ). This makes clear you use one or the other (so not only rely of them being empty; this better explains how these options should be used

Reply:
done.